TombEditor/Controls/DepthBar.cs

```diff
@@ -211,7 +211,7 @@ public bool MouseDown(MouseEventArgs e, Size parentControlSize, Vector2 clickPos
                                             if (mouseDepth <= roomSequences[i][j].MaxDepth && mouseDepth >= roomSequences[i][j].MinDepth)
                                             {
                                                 _roomMouseClicked = roomSequences[i][j].Room;
-                                                _roomMouseOffset = mouseDepth - _roomMouseClicked.Position.Y;
+                                                _roomMouseOffset = mouseDepth - Clicks.FromWorld(_roomMouseClicked.Position.Y);
```

Comment:
I am not fully understanding why `Clicks.From/ToWorld()` methods are used for room coordinates. Was room vertical coordinate format changed as well, together with variable click height?

Reply:
FromWorld -> value / 256 (e.g. 512 / 256 = 2 clicks)
ToWorld -> value * 256 (e.g 2 clicks * 256 = 512 world units)

We store height in world units now exclusively, but some calculations rely on Click height anyway and I preferred not to fiddle with them unless absolutely necessary.